src/components/widget/MasterWidget.js

```diff
@@ -80,6 +101,26 @@ class MasterWidget extends Component {
     clearTimeout(this.timeout);
   }
 
+  getWidgetData = (
+    nextItem,
+    nextIsEditable,
+    nextSupportFieldEdit,
+    nextGetWidgetData
+  ) => {
+    const { widgetData, getWidgetData } = this.props;
+    let { item, isEditable, supportFieldEdit } = this.props;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['supportFieldEdit' is missing in props validation (react/prop-types)](https://app.codacy.com/app/metasfresh/metasfresh-webui-frontend/pullRequest?prid=4689744)